src/YoutubeAtomPlayer.tsx

```diff
@@ -155,6 +170,26 @@ const createOnStateChangeListener = (
         };
     }
 
+    if (event.data === YT.PlayerState.PAUSED) {
+        log('dotcom', {
```

Comment:
Can we remove these sections?

Reply:
Good point, where can we access these logs. looking at the implementation of the `log` it seems it's just doing `console.log` but not sure if it's going into any S3 or something 🤔

Reply:
The `log` function is from `@guardian/libs` which will log messages to the console  but **_only_** if you manually subscribe via:

```js
guardian.logger.subscribeTo('dotcom')
```

I personally find the logs very useful when debugging so I would vote for keeping them.

If you have it switched on the output looks like this:

<img width="1171" alt="youtube-atom-logging-2" src="https://user-images.githubusercontent.com/7014230/154929938-8e2a775e-f8e1-4d93-87e9-5977a4bca775.png">

Reply:
Ah very useful to know, thanks. Haven't come across this before.

Reply:
Ah that's really cool thing